libs/navigation/website/src/lib/navbar/navbar.tsx

```diff
@@ -23,45 +25,172 @@ const appBarStyles = (theme: Theme, isMenuOpen: boolean) =>
 
 export const NavbarInnerWrapper = styled(Toolbar)`
   display: grid;
+  grid-template-columns: auto 1fr auto 1fr auto;
   align-items: center;
   grid-auto-flow: column;
   justify-content: space-between;
   justify-items: center;
   min-height: unset;
+  padding: 0;
 
   ${({theme}) => css`
+    ${theme.breakpoints.up('sm')} {
+      min-height: unset;
+      padding: 0;
+      grid-auto-columns: 1fr;
+    }
+
     ${theme.breakpoints.up('md')} {
       min-height: unset;
-      padding-left: 0;
+      padding: 0;
       grid-auto-columns: 1fr;
     }
   `}
 `
 
-export const NavbarMain = styled('div')`
+const NavbarLinks = styled('div')<{isMenuOpen?: boolean}>`
+  grid-column: 2;
+  margin: 0 ${({theme}) => theme.spacing(1)};
+  display: none;
+  align-items: center;
+  justify-content: flex-start;
+  width: 100%;
+
+  ${({isMenuOpen}) =>
+    isMenuOpen &&
+    css`
+      z-index: -1;
+    `}
+
+  @media (min-width: 740px) {
+    // custom for maximum space usage
+    display: flex;
+  }
+`
+
+const NavbarLink = styled('a')`
+  font-size: 1rem;
+  margin: 0 ${({theme}) => theme.spacing(1)} 0 ${({theme}) => theme.spacing(2)};
+  text-decoration: none;
+  color: ${({theme}) => theme.palette.common.black};
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    font-size: 1.3rem;
+  }
+`
+
+export const NavbarMain = styled('div')<{isMenuOpen?: boolean}>`
   display: flex;
   flex-flow: row wrap;
   align-items: center;
   justify-self: start;
   gap: ${({theme}) => theme.spacing(2)};
+
+  ${({isMenuOpen}) =>
+    isMenuOpen &&
+    css`
+      z-index: -1;
+    `}
+`
+
+export const NavbarActions = styled('div')<{isMenuOpen?: boolean}>`
+  display: flex;
+  flex-flow: row wrap;
+  align-items: center;
+  justify-self: end;
+  gap: ${({theme}) => theme.spacing(2)};
+  grid-column: 5;
+  justify-self: end;
+
+  ${({isMenuOpen}) =>
+    isMenuOpen &&
+    css`
+      z-index: -1;
+    `}
 `
 
 export const NavbarIconButtonWrapper = styled('div')`
   background-color: ${({theme}) => theme.palette.primary.main};
   display: flex;
   justify-content: center;
   align-items: center;
+  width: ${({theme}) => theme.spacing(6.5)};
+  height: ${({theme}) => theme.spacing(6.5)};
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    width: ${({theme}) => theme.spacing(7.5)};
+    height: ${({theme}) => theme.spacing(7.5)};
+
+    svg {
```

Comment:
Any reason we don't pass that down via `css={...}`? Makes it easier to override as it's not using a specificity of 3